pages/about.tsx

```diff
@@ -168,42 +202,73 @@ const AboutPage = () => {
 
       <SectionWrapper header={<H3>The return data</H3>} anchorKey="returndata">
         <p className="pb-8">
-          The return data region is the way a smart contract can return a value
-          after a call. It can be set by external contract calls through the{' '}
+          The return data is the way a smart contract can return a value after a
+          call. It can be set by external contract calls through the{' '}
           <RelativeLink to="#F3" title="RETURN" /> and{' '}
-          <RelativeLink to="#FD" title="REVERT" /> instructions and can be read
+          <RelativeLink to="#FD" title="REVERT" /> instructions, and can be read
           by the calling contract with{' '}
           <RelativeLink to="#3D" title="RETURNDATASIZE" /> and{' '}
           <RelativeLink to="#3E" title="RETURNDATACOPY" />.
         </p>
       </SectionWrapper>
 
-      <SectionWrapper header={<H2>Gas costs</H2>} anchorKey="gascosts">
-        <p className="pb-6">
-          As an incentive to provide resources to run transactions, a fee is
-          paid to send and execute a transaction. The fee is determined by
-          several factors, including the amount of data sent or the amount of
-          work that a transaction requires. That fee is calculated through two
-          mechanisms. The first is a fixed cost that is defined by Ethereum,
-          depending on what is executed. The unit for that is called gas, and
-          remains the same for all transactions, though it can be changed by a
-          hardfork. The second component is the price of one gas unit, which
-          varies over time according to what people are willing to pay to run
-          their transactions. Its unit is ETH per gas.
+      <SectionWrapper header={<H2>Gas Costs</H2>} anchorKey="gascosts">
+        <p className="pb-4">
+          Each transaction on the Ethereum blockchain is vetted by a third-party
+          validator, before it is added to the blockchain. These validators are
+          compensated for conducting this vetting process, and adding
+          transactions to the blockchain, with incentive fee payments. Fees vary
+          from transaction to transaction, contingent on different variables for
+          different forks. Some variables in calculating fees include:
+          <ul className="list-disc mb-2">
+            <br></br>
+            <li>
+              <b>Current price of one gas unit:</b> Gas, or gwei, is a
+              denomination of Ethereum, used to in fee payment. Gas prices vary
+              over time, based on current demand for block space, measured in
+              ETH per gas.
+            </li>
+            <br></br>
+            <li>
+              <b>Calldata size:</b> Each calldata byte costs gas, the larger the
```

Comment:
Add the actual cost calculations?
```
the calldata costs 4 gas per byte equal to 0, and 16 gas for the others (64 before the hardfork <b>Istanbul</b>)
```

Reply:
Not sure what you mean here - this is mentioned in Intrinsic Gas (below)

Reply:
I just expect to see it here. If I come back to get the numbers again, this is where I will look, not the next section.
Just seems weird to not mention it here, when we talk about it.

Reply:
Or at least to put a link to the actual numbers.

Reply:
I can link to the calculations like I did above - feels weird to repeat